### PR TITLE
Revert/460652  perf testing setup

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: app/routes/authenticate.js [hunk 1]
```diff
@@ -25,20 +27,67 @@ module.exports = {
   method: 'GET',
   path: '/authenticate',
   options: {
-    auth: false
+    auth: { mode: 'try' }
   },
   handler: async (request, h) => {
     try {
-      const user = randomise()
+      if (auth.authType === 'dev') {
```
Comment (jbarnsley10) [inline app/routes/authenticate.js:34]: I know this isn't part of your revert, but do we need this anymore? As we have our own OneLogin test accounts, we probably bypass this anyway don't we?

**Review by jbarnsley10 — APPROVED** [review]
LGTM. Would be good to compare this (merged) against the code before the bypass just to be 100% sure you've reverted everything

**Comment by whitewaterdesign** [comment]
> LGTM. Would be good to compare this (merged) against the code before the bypass just to be 100% sure you've reverted everything

totally.  I've done a check with the old PR - and all **seems** ok, but another check would be totally beneficial

File: app/routes/authenticate.js (hunk 1 above)
Comment (whitewaterdesign) [reply app/routes/authenticate.js:34]: Agree we should remove.  Plan to do in a future PR though, as will require updated tests etc.

**Pull request merged**